package.json

```diff
@@ -50,7 +50,8 @@
     "stylelint-config-standard": "^18.2.0"
   },
   "peerDependencies": {
-    "angular": "1.6.10",
-    "angular-ui-bootstrap": "~1.3.3"
+    "angular": "1.7.x",
```

Comment:
To keep consistency across existing applications which is using `@ovh-ux/ng-pagination-front`
I would recommend you to specify: `"angular": "^1.7.5"` (with the `^`).

Here is the actual state (based on the following commit: `1e72016ea69cb3570776cb016e048e9f2373e319`) from `ovh/manager` repository:

- packages/manager/apps/cloud/package.json: `"angular": "^1.7.5",`
- packages/manager/apps/dedicated/package.json: `"angular": "1.7.x",`
- packages/manager/apps/public-cloud/package.json: `"angular": "^1.7.5"`
- packages/manager/apps/telecom/package.json: `"angular": "^1.7.5",`
- packages/manager/apps/web/package.json: `"angular": "^1.6.10",`

I agree that both Dedicated and Web aren't configured like all other applications but in fact they are all running in AngularJS@1.7.8 (from the monorepo).